app/controllers/api/motions_controller.rb

```diff
@@ -11,6 +11,22 @@ def close
     respond_with_resource
   end
 
+  def create_outcome
+    load_and_authorize(:motion, :create_outcome)
+    MotionService.create_outcome(motion: @motion,
+                                 params: permitted_params.motion,
+                                 actor:  current_user)
+    respond_with_resource
+  end
+
+  def update_outcome
+    load_and_authorize(:motion, :update_outcome)
```

Comment:
Interesting. The MotionService already authorises (and raises if action not permitted). Might be cool to have a chat about where we want authorisation to happen. (I've been thinking that keeping it in the service relieves controllers of the task and prevents it being forgotten).